app/views/curriculums/_form.html.erb

```diff
@@ -1,2 +1,110 @@
-<%= f.label :Objetivo_Profesional %>
-<%= f.text_area :professional_objective %>
\ No newline at end of file
+<div class="row">
+  <div class="col s12">
+    <h5>Foto:</h5>
+    <%= f.file_field :profile_picture %>
```

Comment:
Please add the css class for materialize file input styles. There is an example in `company/edit`